packages/office-ui-fabric-react/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -108,13 +108,15 @@ export class MaskedTextField extends BaseComponent<ITextFieldProps, IMaskedTextF
   public render() {
     return (
       <TextField
-        {...this.props}
+        strictMode
```

Comment:
Shouldn't this be disabled by default, even for `MaskedTextField`?

Reply:
Specifically I'm worried that all props are forwarded to `TextField` so even some usages of `MaskedTextField` could potentially break.